GLE_solver.py

```
@@ -1,167 +1,305 @@
+    args = parser.parse_args()
+    
+    # Convert theta0 to radians
+    theta0_rad = args.theta0 * np.pi / 180
+    
+    # Create output directory
+    os.makedirs(args.output_dir, exist_ok=True)
+    
+    # Print parameters
+    print("GLE Solver")
+    print("="*60)
+    print(f"Parameters:")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)